File: fish-web-monitor/src/main/webapp/app/store/parameter/appSystem/FileFormat.js
Ext.define('Eway.store.parameter.appSystem.FileFormat',{
	extend :'Ext.data.Store',
	
	model:'Eway.model.Dict',
	
	data:[{value:'0',display:'JSON'},
	      {value:'1',display:'PROPERTIES'},
	      {value:'2',display:'INI'},
	      {value:'3',display:'XML'}]
});

File: fish-api/fish-api-version/src/main/java/com/yihuacomputer/fish/api/parameter/FileFormat.java
package com.yihuacomputer.fish.api.parameter;

/**
 * @author YiHua
 *
 */
public enum FileFormat {
	JSON("FileForm.JSON",0),
	
	PROPERTIES("FileForm.PROPERTIES",1),
	
	INI("FileForm.INI",2),
	
	XML("FileForm.XML",3);
	
	private int id;
	private String text;
	
	private FileFormat(String text, int id) {
		this.text = text;
		this.id = id;
	}

	public int getId() {
		return id;
	}

	public String getText() {
		return text;
	}

	/**
	 * @param id
	 * @return
	 */
	public static FileFormat getById(int id){
		for(FileFormat each : FileFormat.values()){
			if(each.getId()==id){
				return each;
			}
		}
		throw new IllegalArgumentException(String.format("id=[%id] error", id));
	}
}
